geomesa-fs/geomesa-fs-storage/geomesa-fs-storage-common/src/main/java/org/locationtech/geomesa/fs/storage/common/interop/ConfigurationUtils.java

```diff
@@ -35,6 +36,11 @@ public static void setLeafStorage(SimpleFeatureType sft, boolean leafStorage) {
         sft.getUserData().put("geomesa.fs.leaf-storage", String.valueOf(leafStorage));
     }
 
+    public static void setTargetFileSize(SimpleFeatureType sft, String size) {
+        Memory$.MODULE$.bytes(size).get(); // validate input
```

Comment:
Hmm.... looks like the size bit may be should be factored out into a common place/name (rather than being under "Memory")?

Reply:
i think there may be some semantic differences when discussing size in memory vs disk, but not entirely sure...

Reply:
i didn't think the difference was worth changing anything